includes/admin/functions-menu.php

```diff
@@ -50,18 +50,33 @@ function wpas_tickets_count() {
 
 	global $menu, $current_user;
 
-	if ( current_user_can( 'administrator' )
-		 && false === boolval( wpas_get_option( 'admin_see_all' ) )
-		 || ! current_user_can( 'administrator' )
+	$is_agent = current_user_can( 'administrator' )
```

Comment:
What's the advantage of assigning this to a variable instead of simply using the expression in an `if` statement?

Reply:
$agent_id and the count depend on $is_agent. the cache get/set depend on $agent_id. if you dont store $is_agent, you have to implement the cache retrieval twice